.github/workflows/release-docker.yml

```diff
@@ -19,6 +19,10 @@ jobs:
     runs-on: ubuntu-latest
     environment: Release Docker
     steps:
+      - uses: arduino/setup-protoc@64c0c85d18e984422218383b81c52f8b077404d3 # v1.1.2
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Since we're only building on Docker, I assume this won't be required for the release workflow.

Reply:
arh yea.. you're right!!
https://github.com/DeFiCh/metachain/blob/fd7feb59a7d8cfae821fd612a1182bcdc786232b/Dockerfile#L4